### DISCUSSION TITLE
[Question]: Export as timelapse in 0.14

### DISCUSSION THREAD

**Discussion by Mystically11** [blakeblackshear/frigate#11699] [Ask A Question] [answered]
### What is your question:

Hi

In the 0.14 beta, i've noticed that upon clicking "Export", there are no elements where you can choose to export a time period as a timelapse. Is this feature getting removed?

**Comment by navarrolux** [comment]
I noticed the same thing. I see that the configuration file still accepts the configuration of time-lapse, but I do not see a way to trigger the export of a timelapse on the UI.
Are timelapses available in 0.14?

**Reply by NickM-27** [reply, marked as answer]
only via API, this is mentioned in the release notes / FAQ

**Reply by navarrolux** [reply]
Thank you!
